notebooks/image_models/labs/autoencoder_anomaly_detection.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<ul><li>Add a markdown cell above with a "<strong>Exercise: &lt;description of the task asked&gt;</strong>"</li><li> This seems also too easy. What about instead describing the architecture you want to be implemented in the markdown cell above. For instance something like that:</li></ul><p> </p><p><strong>Exercise:</strong> Complete the <code>build_decoder</code> function below so that that it returns a <code>Sequential</code> with layer architecture to be the exact reverse of the of the encoder layers. <strong>Hint: </strong>You can get the decoder layer information from the encoder object: e.g.   <code>coder.layers[-1].input.shape[1]</code> will return the input shape of the encoder last layer.</p><p> </p><ul><li>Same comment for the encoder exercise above.  </li></ul><p> </p><pre class="ql-syntax" spellcheck="false">
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>